packages/plugin-user/src/user.js

```diff
@@ -164,6 +164,31 @@ const User = SparkPlugin.extend({
   },
 
   /**
+   * Extracts a user object for a user specified by userId(UUID) and organization specified by orgId(UUID)
+   * @param {Object} options
+   * @param {string} options.userId
+   * @param {string} options.orgId
+   * @private
+   * @returns {Promise<Object>}
+   */
+  getUserForOrg(options) {
+    var scimServiceUrl = this.spark.config.device.preDiscoveryServices.scim;
+    if (!options.userId || !options.orgId) {
+     return Promise.reject('Both userId and orgId must be passed in.');
+    }
+    return this.request({
+     method: `GET`,
+     url: scimServiceUrl,
```

Comment:
Use `url` or `uri` when you have a full url; use `service` and `resource` to use a service catalog or prediscovery url.  You're getting "API Definition not found" because, by using `url`, you're hitting `https://identity.webex.com/identity/scim` instead of `https://identity.webex.com/identity/scim/${options.orgId}/v1/Users/${options.userId}`